server/src/test/java/org/apache/druid/server/coordinator/simulate/SegmentLoadingNegativeTest.java

```diff
@@ -91,130 +88,4 @@ public void testImmediateLoadingViolatesThrottleLimit()
     verifyDatasourceIsFullyLoaded(datasource);
   }
 
-  /**
-   * Correct behaviour: The first replica on any tier should not be throttled.
-   * <p>
-   * Fix Apache #12881 to fix this test.
-   */
-  @Test
```

Comment:
Change summary: 
The underlying behaviour for these tests is now fixed!
The corresponding positive test cases can be found in `SegmentLoadingTest`.